onnx/test/automatic_upgrade_test.py

```diff
@@ -1546,6 +1546,9 @@ def test_DFT(self) -> None:
         self._test_op_upgrade(
             "DFT", 17, [[2, 16, 2], []], [[2, 16, 2]], attrs={"inverse": 1}
         )
+        self._test_op_upgrade(
```

Comment:
Is the downgrade to be added post Liqun's related PR?

Reply:
Yes. I will add after it